authapi/tests/test_organizations.py

```diff
@@ -544,6 +560,24 @@ def test_get_teams_for_organization(self):
 
         self.assertEqual(team['id'], team1.id)
 
+    def test_permission_teams_for_organization(self):
+        '''Any member of the organization should be able to see that
+        organization's teams.'''
+        org = SeedOrganization.objects.create()
+        url = reverse('seedorganization-teams-list', args=(org.pk,))
+
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+        user, token = self.create_user()
+        org.users.add(user)
+        team = SeedTeam.objects.create(organization=org)
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + token.key)
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        [resp_team] = response.data
+        self.assertEqual(resp_team['id'], team.pk)
```

Comment:
Would it maybe make sense to have a similar `run_permission_checks` method for reading?